app/api/views/api_view.py

```diff
@@ -0,0 +1,46 @@
+from app.serializers.player_serializer import PlayerSerializer
+from app.models import Player, Spectator, Moderator
+
+from django.http import JsonResponse
+from django.contrib.auth import authenticate, login, logout
+from django.views.decorators.csrf import csrf_exempt
+
+import json
+
+def account_info(request):
+    if not request.user.is_authenticated:
+        return JsonResponse([], status=403, safe=False)
+
+    if request.method == "GET":
+        playerSet = Player.objects.all().filter(user=request.user)
+        if playerSet:
+            player = playerSet[0]
+            serializer = PlayerSerializer(player)
+            return JsonResponse(serializer.data, safe=False)
+        else:
+            return JsonResponse([], status=403, safe=False)
```

Comment:
These status codes look way better and much more fitting. The only change left would be to convert these to helper functions in `utils`. 

That way we can do something like:
```python
return unauthorized(body = [])